File: app/src/main/java/com/erikzuo/portfolioandroidapp/ui/main/education/EducationItemViewModel.java
package com.erikzuo.portfolioandroidapp.ui.main.education;

import android.databinding.ObservableField;

import com.erikzuo.portfolioandroidapp.data.model.Education;

/**
 * Created by Soprano on 9/02/2018.
 */

public class EducationItemViewModel {
    private Education mEducation;

    public ObservableField<String> name;
    public ObservableField<String> degree;

    public EducationItemViewModel(Education education) {
        this.mEducation = education;

        name = new ObservableField<>(mEducation.getName());
        degree = new ObservableField<>(mEducation.getDegree());
    }


}

File: app/src/main/java/com/erikzuo/portfolioandroidapp/ui/main/education/EducationItemViewModel.kt
package com.erikzuo.portfolioandroidapp.ui.main.education

import android.databinding.ObservableField

import com.erikzuo.portfolioandroidapp.data.model.Education

/**
 * Created by Soprano on 9/02/2018.
 */

class EducationItemViewModel(private val mEducation: Education) {

    var name: ObservableField<String?> = ObservableField(mEducation.name)
    var degree: ObservableField<String?> = ObservableField(mEducation.degree)


}
